lib/protoflow-blocks/doc/sys/read_socket.seq.mmd

```diff
@@ -0,0 +1,15 @@
+sequenceDiagram
+    autonumber
+    participant ReadSocket as ReadSocket block
+    participant ReadSocket.output as ReadSocket.output port
+    participant BlockA as Another block
+
+    ReadSocket-->>BlockA: Connect
+
+    loop ReadSocket process
+        ReadSocket->>ReadSocket: Read proto message from TCP port
+        ReadSocket->>BlockA: Message (proto)
+    end
```

Comment:
Let's make these descriptions symmetrical to those in [`ReadStdin`](https://docs.rs/protoflow-blocks/latest/protoflow_blocks/sys/struct.ReadStdin.html):
- "Read bytes from a TCP socket"
- "Message (Bytes)"

Reply:
fixed